pravega/schema-registry
# Issue 227 add type in case it's not provided

Signed-off-by: Shashwat Sharma <shashwat_sharma@dell.com>

**Change log description**  
Currently if the type is not provided with addSchema call, for Protobuf and Avro it's being updated after parsing the file but in case of JSON and Custom it's not implemented and it throws retriable exception and keeps on retrying and ultimately after 1 minute the server returns with http status 504.

**Purpose of the change**  
Fixes #227 

**What the code does**  
Implement similar behavior to Avro and Protobuf and update type if the same has not been provided with the API call addSchema

**How to verify it**  
addSchema of JSON or Custom type without providing the argument "type" but the same should be reflected
